### PR TITLE
Migrating typescript documents to database

### PR DESCRIPTION
**Pull request by adilansari**

### CODE REVIEW

File: docs/typescript/getting-started.mdx [hunk 1]
```diff
@@ -90,7 +106,7 @@ const catalogSchema: TigrisSchema<Catalog> = {
 ```
 
 For detailed documentation on schema representation refer to the
-[data modeling](documents/datamodel) section.
+[data modeling](database/datamodel) section.
 
 ## Create Database and Collection
```
Comment (himank) [inline docs/typescript/getting-started.mdx:111]: Let's break it down, `Create Project` which says that create a project using console or CLI and then have `Create Collection`

Comment (ovaistariq) [reply docs/typescript/getting-started.mdx:111]: For now, skip including the CLI instructions, as they will need to change once the CLI command changes are done.

File: docs/typescript/_snippets/index/_createCollection [hunk 2]
```diff
@@ -1,10 +1,11 @@
 const tigris = new Tigris({
   serverUrl: "api.preview.tigrisdata.cloud",
   clientId: 'xxx',
-  clientSecret: 'xxx'
+  clientSecret: 'xxx',
+  projectName: 'my-ecommerce-project'
 });
 
-const db = await tigris.createDatabaseIfNotExists("catalogdb");
+const db = await tigris.getDatabase();
```
Comment (ovaistariq) [inline docs/typescript/_snippets/index/_createCollection:8]: How about changing this and other schema definitions here to use decorators?

Comment (adilansari) [reply docs/typescript/_snippets/index/_createCollection:8]: That would come next. This change is to only introduce concept of projects and rename the directory. That'd need an update on all examples to have tabbed code snippets for decorators and interface based schema

**Review by ovaistariq — APPROVED** [review]

**Pull request merged**